### PR TITLE
FAQ vaccination_certificate_multiple_scan_possible

### PR DESCRIPTION
**Pull request by dsarkar**
vaccination_certificate_multiple_scan_possible

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -101,6 +101,31 @@
...
+                            "</ul>"
+                        ]
+                    },
+                    {
+                        "title": "Can I save my vaccination certificate on multiple smartphones, e.g., on my private and business phone? How can I transfer the vaccination certificate when changing device? What do I do if I lose my phone? Will the vaccination certificate be included in my backup? Can I manage the certificate in the Corona-Warn-App and the CovPass-App at the same time, or do I have to choose one of the two apps?",
+                        "anchor": "vaccination_certificate_multiple_scan_possible",
+                        "active": true,
+                        "textblock": [
+                            "There is no limit to how often you scan the 2D barcode (QR code) of your vaccination certificates. You can add and manage the vaccination certificates on multiple devices, on a new device, on a device you have reset, and even in several apps. Therefore, a backup is not required."
```
Comment (Ein-Tim) [inline src/data/faq.json:124]:
```suggestion
                            "There is no limit to how often you can scan the 2D barcode (QR code) of your vaccination certificates. You can add and manage the vaccination certificates on multiple devices, on a new device, on a device you have reset, and even in several apps. Therefore, a backup is not required."
```

File: src/data/faq_de.json [hunk 2]
```diff
@@ -102,6 +102,31 @@
...
+                            "</ul>"
+                        ]
+                    },
+                    {
+                        "title": "Kann ich mein Impfzertifikat auf mehreren Smartphones speichern, z.B. auf meinem privaten und dem Geschäftshandy? Wie kann ich die Impfbescheinigung beim Gerätewechsel übertragen? Was mache ich, wenn ich mein Smartphone verloren habe? Ist das Impfzertifikat in meinem Backup enthalten? Kann ich das Zertifikat in der Corona-Warn-App und der CovPass-App gleichzeitig verwalten, oder muss ich mich für eine der beiden Apps entscheiden?",
+                        "anchor": "vaccination_certificate_multiple_scan_possible",
+                        "active": true,
+                        "textblock": [
+                            "Es gibt keine Begrenzung, wie oft Sie den 2D-Barcode (QR-Code) Ihres Impfzertifikats scannen. Sie können die Impfzertifikate auf mehreren Geräten, auf einem neuen Gerät, auf einem zurückgesetzten Gerät, und sogar in mehreren Apps hinzufügen und verwalten. Daher ist auch keine Sicherung (Backup) erforderlich."
```
Comment (Ein-Tim) [inline src/data/faq_de.json:125]:
```suggestion
                            "Es gibt keine Begrenzung, wie oft Sie den 2D-Barcode (QR-Code) Ihres Impfzertifikats scannen können. Sie können die Impfzertifikate auf mehreren Geräten, auf einem neuen Gerät, auf einem zurückgesetzten Gerät, und sogar in mehreren Apps hinzufügen und verwalten. Daher ist auch keine Sicherung (Backup) erforderlich."
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Two small comments here

**Comment by dsarkar** [comment]
@Ein-Tim Thanks

**Pull request merged**